message_ix_models/model/material/data_ammonia_new.py

```diff
@@ -43,7 +45,9 @@ def gen_all_NH3_fert(scenario, dry_run=False):
     }
 
 
-def broadcast_years(df_new, max_lt, act_years, vtg_years):
+def broadcast_years(
+    df_new: pd.DataFrame, max_lt: int, act_years, vtg_years
```

Comment:
What are `act_years` and `vtg_years`? Are these `list[int]` or `pd.Series` or something else?